src/tls_wrap.cc

```diff
@@ -589,8 +590,13 @@ int TLSWrap::DoWrite(WriteWrap* w,
     return 0;
   }
 
-  if (ssl_ == nullptr)
+  if (ssl_ == nullptr) {
+    static char msg[] = "Write after DestroySSL";
+    char* tmp = new char[sizeof(msg)];
+    memcpy(tmp, msg, sizeof(msg));
+    error_ = tmp;
```

Comment:
I'd add a `CHECK_EQ(error_, nullptr)`, otherwise it's a memory leak waiting to happen.